### ISSUE TITLE
Incompatibility of plugin version 1.4.1 with version 5.5.1.4

### ISSUE THREAD

**Issue by kotenok2000** [maxisoft/ASFFreeGames#54]
**Describe the bug**
![image](https://github.com/maxisoft/ASFFreeGames/assets/8822292/092b7f85-9b31-4f1b-a4f6-bf9911e84cf3)

**To Reproduce**
install plugin version 1.4.1 on asf 5.5.1.4

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by pjlalvarenga** [comment]
Same for me!
To Reproduce
install plugin version 1.4.1 on asf 5.5.1.4 (Both Generic and x64)

Expected behavior
should load but both are interrupted by exception and closes

Screenshots
Same as user's

Desktop (please complete the following information):

OS: Windows 11 Education
Browser: Chrome (but running without browser)
Version 1.4.1 asf v5.5.1.4

Additional context
Never tried the plugin before, so I don't know if it used to work in previous version combinations or when it started to have this error.

**Comment by thierrybla** [comment]
Anyone that has a workaround?

**Comment by AndyShade** [comment]
i use https://github.com/Citrinate/FreePackages for now

**Comment by OzymoGit** [comment]
> i use https://github.com/Citrinate/FreePackages for now

is FreePackages the same as this poject

**Comment by existful** [comment]
Nop, 
there are different:

[ASFFreeGames](https://github.com/maxisoft/ASFFreeGames/): Checks/Scans every 30 min reddit for new free games.
[FreePackages](https://github.com/Citrinate/FreePackages):  Finds free packages on Steam and adds them to your account.

**Comment by OzymoGit** [comment]
> Nop, there are different:
> 
> [ASFFreeGames](https://github.com/maxisoft/ASFFreeGames/): Checks/Scans every 30 min reddit for new free games. [FreePackages](https://github.com/Citrinate/FreePackages): Finds free packages on Steam and adds them to your account.

do you have any recommend config for Freepackage?

**Comment by PhasecoreX** [comment]
This is how I set mine up:
```
  "EnableFreePackages": true,
  "FreePackagesFilters": [
    {
      "Types": [
        "DLC"
      ]
    },
    {
      "Types": [
        "Game"
      ],
      "Languages": [
        "english"
      ],
      "MinReviewScore": 6,
      "IgnoreFreeWeekends": true,
      "NoCostOnly": true
    }
  ]
```
It'll grab all missing free DLC for games in your account, as well as all English language games with a positive rating that aren't just a weekend trial and aren't an "always free" type game.

That last option (NoCostOnly true) was important for me, as without it, it'll eventually add every free to play game, which a vast majority of them do not give out cards and don't count towards total game count.

I'm using mine for actually getting decent games to play, but for purely card collection you could remove the language filter and the min review score. I don't think you can get cards from free weekend games though, so leave that. I guess at that point you could also remove the whole DLC part too:
```
  "EnableFreePackages": true,
  "FreePackagesFilters": [
    {
      "Types": [
        "Game"
      ],
      "IgnoreFreeWeekends": true,
      "NoCostOnly": true
    }
  ]
```

**Comment by maxisoft** [comment]
Hey everyone,

Just fixed the issue with plugin version 1.4.1 not working with ASF 5.5.1.4 (issue #54). The new update in PR #65 sorts it all out.

Thanks to everyone who reported this and helped out by sharing info. Update to version 1.5.1 and you should be good to go.

Cheers,
Maxisoft
